src/util/vpnDetection.ts

```diff
@@ -61,14 +61,17 @@ const isVPN = async (ip: string): Promise<boolean> => {
   console.log(`VPN Cache size: ${vpnCache.size}`);
 
   // Must pass both vpn checks
-  const result = (await isVpnCheck1(ip)) || (await isVpnCheck2(ip));
+  const VPNCheck1 = await isVPNCheck1(ip);
+  const VPNCheck2 = await isVPNCheck2(ip);
```

Comment:
Can you move the logging up to here. I prefer it being one line. We don’t expect other files to use these functions.

Reply:
If we want to restrict usage, we could make it a class and make those private methods but let’s not worry about those :)